### ISSUE TITLE
[RTL] The overlays do not disappear when the table goes out of the viewport

### ISSUE THREAD

**Issue by budnix** [handsontable/handsontable#9233]
### Description

The overlays do not disappear when the table goes out of the viewport. This happens only in a case when Handsontable is initialized without sizing (the Window object is a scrollable element).

The issue:
![Kapture 2022-02-23 at 08 15 22](https://user-images.githubusercontent.com/571316/155275517-dcb0d205-1686-4371-ac6d-c1befb66cde7.gif)

For reference, this is how it works in 11.1.0:
![Kapture 2022-02-23 at 08 16 37](https://user-images.githubusercontent.com/571316/155275661-cf74351d-9dc6-4664-aaec-c5b790bc1f2c.gif)

The same happens for the inline-start overlay.

### Configuration
```html
  <style>
    body {
      padding: 5px;
      margin: 50px;
      margin-top: 300px;
      margin-bottom: 3000px;
      margin-right: 3000px;
      padding-right: 3000px;
    }
  </style>
```
```js
{
  data: Handsontable.helper.createSpreadsheetData(100, 25),
  fixedRowsTop: 2,
  fixedRowsBottom: 2,
  colHeaders: true,
  rowHeaders: true,
}
```

**Comment by aninde** [comment]
Fixed with v12.0.0 https://jsfiddle.net/qxjtovmb/2/.

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
